### PR TITLE
Add a LICENCE file

### PR DESCRIPTION
**Pull request by WhyNotHugo**

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2015-2017 Shiftgig Inc.
```
Comment (TylerHendrickson) [inline LICENCE]: I like the practice of putting the license type as the first line of the `LICENSE` file (followed by a blank line).  At the very least, it provides easy reference (for us as well as anyone looking for examples).

Comment (TylerHendrickson) [inline LICENCE]: Missing comma- should be
```
Copyright (c) 2015-2017, Shiftgig Inc.
```

**Review by TylerHendrickson — CHANGES_REQUESTED** [review]
@hobarrera Two quick changes for you

File: LICENCE (hunk 1 above)
Comment (WhyNotHugo) [reply LICENCE]: I haven't seen that before, but I like the idea since it makes recognizing it *a lot* easier.

**Comment by WhyNotHugo** [comment]
Both things fixed.

**Review by TylerHendrickson — APPROVED** [review]

**Review by TylerHendrickson — APPROVED** [review]

**Pull request merged**